src/platform/renesas/CHIPPlatformConfig.h

```diff
@@ -0,0 +1,33 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2019 Google LLC.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ * @file
+ * Platform-specific configuration overrides for CHIP.
+ */
+
+#pragma once
+
+#define CHIP_CONFIG_MAX_FABRICS 5
```

Comment:
This is also in CHIPProjectConfig.h and it this is more an application settings. The connection pool size I'm not sure about, but provisioning as well. 

I know these application settings cause a bit of a circular dependency, but I think that's better than double bookkeeping. Is there a way to include/inject that file instead?

Reply:
Removed all unneeded configs leaving an empty config file. Unfortunately I cannot remove the config file as it is always included:

``` 
    TSBM-303 Removed unneeded configs:

    - CHIP_CONFIG_MAX_FABRICS application specific, was already in trust repo config files
    - CHIP_CONFIG_UNAUTHENTICATED_CONNECTION_POOL_SIZE Since none of the other platfomr set it
       except for 1, I think the default of 4 is sufficient
    - CHIP_DEVICE_CONFIG_ENABLE_JUST_IN_TIME_PROVISIONING is unused
    - CHIP_DEVICE_CONFIG_ENABLE_THREAD_COMMISSIONABLE_DISCOVERY We don't have thread, and default is false.

    CHIP_LOG_FILTERING is defined in application config file in trust repo.

Reply:
Fixed in other https://github.com/meulemanelectronics/connectedhomeip/pull/22